### PR TITLE
created forgot password page

### PR DESCRIPTION
**Pull request by kgajriaAgarwal**
i have created forgot password page.Kindly review it and let me know what do you think about it !
image
image

### CODE REVIEW

File: Authentication/ForgotPassword/ForgotPassword.css [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    height: 100%;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+}
+
+section .login-content-container .login-form{
+    width:50%;    
+}
+
```
Comment (Ankur9669) [inline Authentication/ForgotPassword/ForgotPassword.css:52]: Suggestion: If possible reduce the specificity of the selectors

File: Authentication/ForgotPassword/ForgotPassword.css [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+.Blithe-logo{
+    height:5rem;
```
Comment (Ankur9669) [inline Authentication/ForgotPassword/ForgotPassword.css:54]: fix: name should be blithe-logo

File: Authentication/ForgotPassword/ForgotPassword.css [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    margin-bottom: 2rem;
+    text-align: left;
+}
+
+section .login-content-container .login-form .btn-primary{
+    width:50%;
+}
+
+@media (max-width:768px){
+    section .login-img-container{
```
Comment (Ankur9669) [inline Authentication/ForgotPassword/ForgotPassword.css:80]: fix: Use @media only screen and (max---) syntax

**Review by Ankur9669 — COMMENTED** [review]
The UI looks good, but you have user very specific selectors if possible try to reduce specificity

**Review by mehulsatardekar — APPROVED** [review]
do the changes whatever @Ankur9669  has suggested other than good to merge this branch

File: Authentication/ForgotPassword/ForgotPassword.css (hunk 2 above)
Comment (kgajriaAgarwal) [reply Authentication/ForgotPassword/ForgotPassword.css:54]: done!

File: Authentication/ForgotPassword/ForgotPassword.css (hunk 3 above)
Comment (kgajriaAgarwal) [reply Authentication/ForgotPassword/ForgotPassword.css:80]: I think it doesn't makes any differnce doing this change

File: Authentication/ForgotPassword/ForgotPassword.css (hunk 1 above)
Comment (kgajriaAgarwal) [reply Authentication/ForgotPassword/ForgotPassword.css:52]: ok

**Pull request merged**